src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -188,18 +201,6 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Defa
         ): null;
     }
 
-    protected get proteinImpactTypePanel(): JSX.Element | null
```

Comment:
now we are using `ProteinImpactTypeBadgeSelector` component to enable multiple filtering